packages/parse5/lib/tokenizer/index.ts

```diff
@@ -536,6 +542,7 @@ export class Tokenizer {
             if (this.currentCharacterToken.type !== type) {
                 this.currentLocation = this.getCurrentLocation(0);
                 this._emitCurrentCharacterToken(this.currentLocation);
```

Comment:
We can't drop the parsed chunk in `_emitCurrentCharacterToken`, as it is called from `prepareToken` and would remove the token right afterwards.